### DISCUSSION TITLE
PLL not locked?

### DISCUSSION THREAD

**Discussion by jruben4** [merbanan/rtl_433#2987] [General]
I had this docker working with unraid and a Rafael Micro R820T tuner for 433 mhz traffic. Suddenly I am getting an error
[R82XX] PLL not locked!

This might have happened when I updated to the newest version of unraid (6.12.10), not sure. I tried updating the DVB drivers but that didn't change anything.

**Comment by zuckschwerdt** [comment]
That's not an error, ignore it.

**Reply by jruben4** [reply]
It doesn't seem to be hearing any traffic anymore, it used to work fine.

**Reply by kalbasit** [reply]
@jruben4 I'm seeing the same thing, were you able to fix your issue?

**Reply by zuckschwerdt** [reply]
Try to add `-Y autolevel -M level -M noise` to adjust and visualize the received levels.

**Reply by RCasas20** [reply]
I know this post have almost 2 years, I had the same problem with a RTL-SDR Dongle recently and basically what I did was update drivers and change the USB port where was connected. Since PLL is not locked, at the time you syntonize some frequency you can't get anything where you was able before.

My problems begun when trying to check short wave frequencies and dongle get stucked 


Some articles of interest
https://www.analog.com/en/resources/analog-dialogue/articles/phase-locked-loop-pll-fundamentals.html
https://www.rtl-sdr.com/rtl-sdr-quick-start-guide/
https://github.com/rtlsdrblog/rtl-sdr-blog
